docs/modules/operation/pages/database-reports/templates/event.adoc

```diff
@@ -1,22 +1,60 @@
 [[event-report]]
 = Event Analysis Report
 
-Analyse events based on event source and quantity by nodes.
+
+== Report Contents
```

Comment:
```suggestion

```

Reply:
Should we remove all the "Report Contents" on the other pages?